Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN bash -ex install.sh 2>&1 && \
 ARG PHP_ENV_CONFIG
 ENV PHP_ENV_CONFIG /etc/lighttpd/conf-enabled/15-fastcgi-php.conf
 ARG PHP_ERROR_LOG
-ENV PHP_ERROR_LOG /var/log/lighttpd/error.log
+ENV PHP_ERROR_LOG /var/log/lighttpd/error-pihole.log
```

Comment:
`ACCESS_LOG` is missing in general. Should this be added?

Reply:
To be honest, I don't think this one is actually even used anywhere.

Reply:
It's only used by `lighttpd` to log each request, but I don't think it is used elsewhere.